pkg/controller/manager/manager_controller_test.go

```diff
@@ -772,5 +794,64 @@ var _ = Describe("Manager controller tests", func() {
 				Expect(instance.Status.Conditions[2].ObservedGeneration).To(Equal(generation))
 			})
 		})
+
+		It("Multi-tenant/namespaced reconciliation", func() {
```

Comment:
I think we should put multi-tenant tests within their own Context() to separate them from the single tenant tests.

We should also follow the normal `It()` clause conventions starting with "should" - e.g., 

```
It("should reconcile a namespaced manager instance")
```